packages/react-sdk/src/components/ElementBar/FontFamilyButton/FontFamilyButton.tsx

```diff
@@ -50,7 +50,7 @@ export function FontFamilyButton() {
       size="small"
       variant="standard"
       disableUnderline={true}
-      value={activeFontFamily}
+      value={activeFontFamily ?? 'Inter'}
```

Comment:
I don't really like this. The type promises, that there is always a font family set. See [here](https://github.com/nordeck/matrix-neoboard/blob/535f49872de98979cfca71245c134f96028808f3/packages/react-sdk/src/components/Layout/useLayoutState.tsx#L117).

We should fix this at the place where we set the active font family. There should never be an undefined value in the layout state in the first place.

Reply:
This has been done